### ISSUE TITLE
`Graphic` class needs to be rewritten

### ISSUE THREAD

**Issue by anthonykulis** [anthonykulis/automobile#17]
https://github.com/anthonykulis/automobile/blob/master/edu/jalc/automobile/parts/Graphic.java#L1

2 points

**Comment by anthonykulis** [comment]
i think this can simply have the attribute of `graphic` (which is same as class name), refactored to something like `name`

**Comment by KenanJones** [comment]
I have added a constructor to Graphic in my repo that takes a String parameter. since it overloads the constructor, it shouldn't mess anyone else up.

**Comment by anthonykulis** [comment]
No. If you are going to handle issues existing on automobile, you need to
make a separate fork and pr for that issue. If you commit what is in your
repo, you will break everyone elses.

**Comment by KenanJones** [comment]
it is included in the pr I just made
